packages/api/db/migration/20221114230005_alter_field_work_task_table.js

```diff
@@ -0,0 +1,97 @@
+// import fieldWorkData from '../../db/seeds/seedData/field_work.json';
+
+export const up = async function (knex) {
+  const fieldWorkData = [
+    {
+      field_work_type_translation_key: 'COVERING_SOIL',
+      field_work_name: 'Covering soil',
+    },
+    {
+      field_work_type_translation_key: 'FENCING',
+      field_work_name: 'Fencing',
+    },
+    {
+      field_work_type_translation_key: 'PREPARING_BEDS_OR_ROWS',
+      field_work_name: 'Preparing beds or rows',
+    },
+    {
+      field_work_type_translation_key: 'PRUNING',
+      field_work_name: 'Pruning',
+    },
+    {
+      field_work_type_translation_key: 'SHADE_CLOTH',
+      field_work_name: 'Shade cloth',
+    },
+    {
+      field_work_type_translation_key: 'TERMINATION',
+      field_work_name: 'Termination',
+    },
+    {
+      field_work_type_translation_key: 'TILLAGE',
+      field_work_name: 'Tillage',
+    },
+    {
+      field_work_type_translation_key: 'WEEDING',
+      field_work_name: 'Weeding',
+    },
+  ];
+  await knex.schema.createTable('field_work', function (t) {
```

Comment:
@anant00729 

Consider naming table 'field_work_type'

Reply:
the refactored tables will look like this

<img width="769" alt="Screen Shot 2022-11-29 at 10 01 36 AM" src="https://user-images.githubusercontent.com/20675885/204609284-bfc5d9b8-5ce9-4f0a-88c5-3cc7b8f91a74.png">

Reply:
Are you changing the current table naming convention?

![image](https://user-images.githubusercontent.com/30075973/204636907-cb153ae2-e91c-45a8-933f-1a86af463b47.png)

![image](https://user-images.githubusercontent.com/30075973/204636793-30124a1d-6951-490d-abf2-21fcbc45d141.png)